DeusEx/Classes/BalanceFireExtinguisher.uc

```diff
@@ -0,0 +1,18 @@
+class BalanceFireExtinguisher injects FireExtinguisher;
+
+function Timer()
+{
+    UseOnce();
+}
+
+simulated function PreBeginPlay()
+{
+    Super.PreBeginPlay();
+    maxCopies=default.maxCopies;
+}
+
+defaultproperties
+{
+    maxCopies=5
```

Comment:
this is actually a nerf for Stick With the Prod mode, because you can't carry multiple stacks